### DISCUSSION TITLE
Pause / Suspend is it safe?

### DISCUSSION THREAD

**Discussion by Zodler** [staxrip/staxrip#1301] [Q&A]
Is suspending an encode and later resuming it, safe? I remember sometime in the past I had done that and the resulting file had video artifacts at the moment of pause.

**Comment by Dendraspis** [comment]
Whenever I tested it in the past there where no problems with the outcome.  🤔

**Comment by jessielw** [comment]
To my understanding, suspend still holds the current state as is in memory, meaning you'll gain a portion of the resources back, but the actual job is still there, looping, just being held in the current place. 

So, while you can do this, you should just ensure you are encoding at a time when you don't need the resources for something else tbh.
